uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-fs/src/main/java/org/uberfire/java/nio/fs/file/SimpleFileSystemProvider.java

```diff
@@ -115,14 +113,12 @@ public String getScheme() {
     }
 
     @Override
-    public FileSystem getFileSystem(final URI uri)
-            throws IllegalArgumentException, FileSystemNotFoundException, SecurityException {
+    public FileSystem getFileSystem(final URI uri) {
```

Comment:
why not preserve the original throws? this was more for `documentation` purposes

Reply:
Well, I do 100% agree with you, but the sonarCloud flags it as a rule violation.

Reply:
I think in this case is best ignore sonarCloud. Not sure who's reponsible for the sonarCloud rules, but in this case I'd just ignore.

Reply:
Reverted back to original style, but SonarCloud might not be happy :(